src/main/java/com/googlesource/gerrit/plugins/oauth/GitLabApi.java

```diff
@@ -0,0 +1,161 @@
+// Copyright (C) s/2015/2017 The Android Open Source Project
```

Comment:
Sorry, for misundeerstanding, I meant just 2017, s/... for sed command, meaning/aking to replace your original version with 2017 (current year)


    // Copyright (C) 2017 The Android Open Source Project